cluster-operator/src/test/java/io/strimzi/operator/cluster/model/KafkaClusterTest.java

```diff
@@ -34,7 +35,7 @@
     private final String metricsCmJson = "{\"animal\":\"wombat\"}";
     private final String configurationJson = "{\"foo\":\"bar\"}";
     private final ConfigMap cm = ResourceUtils.createKafkaClusterConfigMap(namespace, cluster, replicas, image, healthDelay, healthTimeout, metricsCmJson, configurationJson);
-    private final KafkaCluster kc = KafkaCluster.fromConfigMap(cm);
+    private final KafkaCluster kc = KafkaCluster.fromDescription(cm, Collections.EMPTY_LIST);
```

Comment:
Minor point of style: I usually use `Collection.emptyList()`, which lets the compiler infer the type arguments for the resulting list, and avoids raw types leaking into the code.